server/src/main/scala/org/dbpedia/extraction/server/resources/TemplateStatistics.scala

```diff
@@ -170,7 +171,7 @@ class   TemplateStatistics(@PathParam("lang") langCode: String, @QueryParam("p")
             var redirectMsg = ""
             for (redirect <- targetRedirect) {
               if (mappingStat.isMapped) {
-                  //redirectMsg = " NOTE: the mapping for " + wikiDecode(redirect, language).substring(createMappingStats.templateNamespace.length()) + " is redundant!"
+                  redirectMsg = " NOTE: the mapping for " + redirect.substring(manager.templateNamespace.length) + " is redundant!"
```

Comment:
Just tested this locally and looks quite good! what would be nice is to make the title a link.
We also use bootstrap css and enclosing this into an alert or a warning would look better
http://getbootstrap.com/2.3.2/components.html#alerts